convert2rhel.ini.example

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
```

Comment:
I don't think the `.example` suffix of the file is necessary. It just gives the user another barrier that does not add much value. It can be actually counter-productive in case we want to introduce some default values that might be possible to change by the user.